src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -4517,6 +4657,33 @@ public class AddressAdminClient implements BackgroundResource {
    * }
    * </code></pre>
    *
+   * @param region Name of the region for this request.
+   * @param address A reserved address resource.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   */
+  @BetaApi
+  public final Operation addressesInsert(RegionName region, Address address) {
+
+    AddressesInsertHttpRequest request =
+        AddressesInsertHttpRequest.newBuilder()
+        .setRegionWithRegionName(region)
+        .setAddress(address)
+        .build();
+    return addressesInsert(request);
+  }
+
+  // AUTO-GENERATED DOCUMENTATION AND METHOD
+  /**
+   * Creates an address resource in the specified project using the data included in the request.
+   *
+   * Sample code:
+   * <pre><code>
+   * try (AddressAdminClient addressAdminClient = AddressAdminClient.create()) {
+   *
+   *   Operation response = addressAdminClient.addressesInsert();
```

Comment:
There is a TODO in InitCodeTransformer for filling out the parameters of the sample method call. It's more work than I'd like to get into right now.